src/material-experimental/mdc-table/_table-theme.scss

```diff
@@ -53,10 +53,8 @@
 
 @mixin density($config-or-theme) {
   $density-scale: theming.get-density-config($config-or-theme);
-  .mat-mdc-table {
-    @include mdc-data-table-theme.density($density-scale,
-      $query: mdc-helpers.$mat-base-styles-query);
-  }
+  @include mdc-data-table-theme.density($density-scale,
```

Comment:
Isn't this going to cause conflicts with other MDC-based tables on the same page?